src/localization.tsx

```diff
@@ -53,6 +53,9 @@ type SetLanguageFunc = (language: string | null) => any;
  */
 type TFunc<T> = (key: Leaves<T>, context?: Record<string, any>) => string;
 
+type THasKeyFunc<T> = (key: Leaves<T>) => boolean;
```

Comment:
Here we should allow to pass any `string` instead of `Leaves<T>`

Reply:
makes sense. I changed that in the next commit.